### PR TITLE
Extension method implemented

### PR DESCRIPTION
**Pull request by simazm8**

### CODE REVIEW

File: Gariunai Cloud Services/Account/AccountForm.cs [hunk 1]
```diff
@@ -24,8 +24,10 @@ public AccountForm(Form previousForm)
             if (currentUser.Image != null)
                 ovalPictureBox1.Image = ByteArrayToImage(currentUser.Image);
             else
+            {
                 ovalPictureBox1.ImageLocation =
                     Path.Combine(Application.StartupPath, "Resources\\empty-profile-picture-png-2.png");
+            }
```
Comment (accountas) [inline Gariunai Cloud Services/Account/AccountForm.cs:30]: surround either both one-line statements with {} or none

File: Gariunai Cloud Services/StringExtension.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+            var current = 0;
+            var output = new char[input.Length];
+            var skipped = false;
+
+            foreach (var c in input.ToCharArray())
+            {
+                if (char.IsWhiteSpace(c))
+                {
+                    if (!skipped)
```
Comment (accountas) [inline Gariunai Cloud Services/StringExtension.cs:20]: can be combined into a single if statement

File: Gariunai Cloud Services/LogIn/CreateNewForm.cs [hunk 3]
```diff
@@ -13,23 +13,27 @@ public CreateNewForm()
...
-            if (state.Item1)
+            if (state)
             {
                 label1.ForeColor = Color.Green;
-                label1.Text = state.Item2;
+                label1.Text = message;
             }
-
-            if (!state.Item1)
+            else if(!state)
```
Comment (accountas) [inline Gariunai Cloud Services/LogIn/CreateNewForm.cs:29]: wrote a comment on previous pull request

**Review by accountas — APPROVED** [review]

**Review by accountas — APPROVED** [review]

File: Gariunai Cloud Services/StringExtension.cs (hunk 2 above)
Comment (simazm8) [reply Gariunai Cloud Services/StringExtension.cs:20]: Actually no, because then else triggers when it's not supposed to.

Comment (accountas) [reply Gariunai Cloud Services/StringExtension.cs:20]: yes, my bad

**Pull request merged**